parsing/src/parts.ts

```diff
@@ -17,15 +18,17 @@ function getItems(data: any[]): PartDataInterface {
                 parts["NuclearWaste"] = { 
                     name: "Uranium Waste",
                     stackSize: 500, //SS_HUGE
-                    isFluid: false,
-                    isFicsmas: false,
+                    isFluid: isFluid("NuclearWaste"),
```

Comment:
Seems a little pointless to call the isFluid function here if it's hard coded data?

Reply:
it's half a refactor. This manual insert will hopefully be going away soon in favor of the whitelist (which will extract the details).